index.js

```diff
@@ -182,14 +184,15 @@ Bankai.prototype.serviceWorker = function (cb) {
   })
 }
 
-Bankai.prototype.assets = function (edgeName, cb) {
-  assert.equal(typeof edgeName, 'string')
+Bankai.prototype.assets = function (filename, cb) {
+  assert.equal(typeof filename, 'string')
   assert.equal(typeof cb, 'function')
   var stepName = 'assets'
   var self = this
   this.queue[stepName].add(function () {
-    var data = self.graph.data[stepName][edgeName]
-    if (!data) return cb(new Error('bankai.asset: could not find a file for ' + edgeName))
+    filename = path.join(self.dirname, filename)
+    var data = self.metadata.assets[filename]
+    if (!data) return cb(new Error('bankai.asset: could not find a file for ' + filename))
```

Comment:
Why is filesystem metadata being returned here?
Shouldn't this return just `filename`? See above where `http.js` expects to get a filename that it can serve.

Suggested fix:
```js
filename = path.join(self.dirname, filename)
cb(null, filename)
```